Documents/Automation/ResizeInstance/Documents/aws-ResizeInstance.json

```diff
@@ -47,6 +47,7 @@
     {
       "name": "stopInstances",
       "action": "aws:changeInstanceState",
+      "onFailure": "step:deleteCloudFormationTemplate",
```

Comment:
I agree on this specific onFailure behavior, but what about failures that occur at the **resizeInstance** step (i.e. due to insufficient IAM permissions) or the **startInstances** step (i.e. limit increase needed to start the instance with the new instance type)?

I am of the opinion that in any failure case of this doc we would want to cleanup the temporary stack that was created. This is in general true for any document that follows this specific design pattern (createStack --> Lambda --> deleteStack).

What do you think?

Reply:
@alemartini I think this is a good idea (and something I should've done since the beginning anyway). Fixed.